patches/tModLoader/Terraria/Localization/Content/pl-PL/tModLoader.json

```diff
@@ -195,26 +194,26 @@
 		// "MBExceededBandwidth": "The Mod Browser server has exceeded its daily bandwidth allotment. Please consult this mod's homepage for an alternate download or try again later.",
 		// "MBUnknownMBError": "Unknown Mod Browser Error. Try again later.",
 		// "MBNoModsFound": "No mods found",
-		// "MBBrowserError": "The mod browser encountered an error.",
-		// "XSecondsAgo": "{0} {^0:second;seconds} ago",
-		// "XMinutesAgo": "{0} {^0:minute;minutes} ago",
-		// "XHoursAgo": "{0} {^0:hour;hours} ago",
-		// "XDaysAgo": "{0} {^0:day;days} ago",
-		// "XMonthsAgo": "{0} {^0:month;months} ago",
-		// "XYearsAgo": "{0} {^0:year;years} ago",
+		"MBBrowserError": "Przeglądarka modów napotkała błąd.",
+		"XSecondsAgo": "{0} {^0:sekunda;sekund} temu",
+		"XMinutesAgo": "{0} {^0:minuta;minut} temu",
+		"XHoursAgo": "{0} {^0:godzina;godzin} temu",
+		"XDaysAgo": "{0} {^0:dzień;dni} temu",
+		"XMonthsAgo": "{0} {^0:miesiąc;miesiące} temu",
+		"XYearsAgo": "{0} {^0:rok;lata} temu",
```

Comment:
Polish has 3 plural grammatical cases:
![image](https://user-images.githubusercontent.com/6795251/119062579-cd692e00-b9df-11eb-9ba0-706c26a318d1.png)
So you need to provide all of them, like: `{^0:plural 0;plural 1; plural 2}`.

Reply:
Sorry I didn't notice that.  Pushing commit right now.